templates/server/puppetserver/conf.d/ca.conf.erb

```diff
@@ -0,0 +1,22 @@
+# CA-related settings
+certificate-authority: {
+
+    # settings for the certificate_status HTTP endpoint
+    certificate-status: {
+
+        # this setting contains a list of client certnames who are whitelisted to
+        # have access to the certificate_status endpoint. Any requests made to
+        # this endpoint that do not present a valid client cert mentioned in
+        # this list will be denied access.
+<%- if scope.lookupvar('puppet::server_ca_authorization_required') == false -%>
+        authorization-required: false
+<%- else -%>
+        authorization-required: true
+        client-whitelist: [
+<%- scope.lookupvar('puppet::server_ca_client_whitelist').each do |client| -%>
+            <%= client %>,
+    <%- end -%>
+<%- end -%>
+        ]
```

Comment:
This needs to be moved up a line or it break with `puppet::server_ca_authorization_required` set to False.